server/src/utils/server.js

```diff
@@ -49,4 +49,21 @@ const serverOnListening = server => {
   console.log("Listening on " + bind);
 };
 
-export { serverNormalizePort, serverOnError, serverOnListening };
+// Generate Session Configuration that will be used on server instantiation
+const createSessionConfigForEnv = envType => {
+  const sessionSecret = envType == 'dev' ? 'Insecure Dev Secret' : process.env.SESSION_SECRET;
```

Comment:
you're insecure... don't know what for.. https://www.youtube.com/watch?v=fuGDrAZ9JyM

Reply:
You're turning heads when you walk in the do-oh-ooor?